types/mini-css-extract-plugin/mini-css-extract-plugin-tests.ts

```diff
@@ -41,15 +42,16 @@ configuration = {
             // or any other compile-to-css language
             {
                 test: /\.less$/,
-                use: [MiniCssExtractPlugin.loader, 'css-loader', 'style-loader'],
+                use: [MiniCssExtractPlugin.loader, "css-loader", "style-loader"],
             },
             // You could also use other loaders the same way. I. e. the autoprefixer-loader
         ],
     },
     // Use the plugin to specify the resulting filename (and add needed behavior to the compiler)
     plugins: [
         new MiniCssExtractPlugin({
-            filename: '[name].css',
+            filename: "[name].css",
+            experimentalUseImportModule: true,
```

Comment:
This should be it's own test. If you look at the rest of the file, you'll see we have a block for each option. 
That way this base test always asserts it can be undefined

Reply:
amended, thx!